src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(Color.values().length)].name();
```

Comment:
Better split it into two lines. It would be easier to read
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```